spec/11-statements.md

```diff
@@ -308,13 +291,21 @@ else  // this else does go with the outer if
   <i>case-statements:</i>
     <i>case-statement</i> <i>statement-list<sub>opt</sub></i> <i>case-statements<sub>opt</sub></i>
     <i>default-statement</i> <i>statement-list<sub>opt</sub></i> <i>case-statements<sub>opt</sub></i>
+
+  <i>case-statement:</i>
+    case   <i>expression   case-default-label-terminator   statement</i>
+
+  <i>default-statement:</i>
+    default  <i>case-default-label-terminator   statement</i>
```

Comment:
It's already an existing issue, but these rules don't looks right to me. The `statement-list_opt` part of `case-statements` looks wrong to me. Instead the `statement` in `case-statement`/`default-statement` should be `statement-list_opt`.